deltachat-ios/Helper/MediaPicker.swift

```diff
@@ -77,19 +77,10 @@ class MediaPicker: NSObject, UINavigationControllerDelegate {
     func showDocumentLibrary(selectFolder: Bool = false) {
         let documentPicker: UIDocumentPickerViewController
         if selectFolder {
-            if #available(iOS 15.0, *) {
-                documentPicker = .init(forOpeningContentTypes: [UTType.archive], asCopy: false)
-            } else {
-                documentPicker = .init(documentTypes: [kUTTypeArchive] as [String], in: .open)
-            }
+            documentPicker = .init(forOpeningContentTypes: [UTType.archive], asCopy: false)
```

Comment:
not totally sure, would be cool if someone double checks:

the `if #available(iOS 15.0, *) {` might always be wrong and should have always been `if #available(iOS 14.0, *) {`.

Xcode gives me a warning otherwise that `documentPicker = .init(documentTypes: [kUTTypeArchive] as [String], in: .open)` is deprecated in iOS 14 (the warning was probably not there before as we deployed to iOS 13)

Reply:
As, according to [The Docs](https://developer.apple.com/documentation/uikit/uidocumentpickerviewcontroller/3566733-init)™, that initializer is available on iOS 14, so it smells like a misunderstanding, not to say wrong.

Maybe it has [something to do with Documents](https://github.com/deltachat/deltachat-ios/commit/fc88a075f569db6a0238f8436c2dae91a742d73a), so that this Deltachat-feature was made available to iOS 15+, only?

Reply:
I think this needs to be checked for all the kUTType strings? They are deprecated in iOS 14 and I think we'll get warnings eg in `ChatDropInteraction.dropInteraction(performDrop:)`

Reply:
I think this needs to be checked for all the kUTType strings? They are deprecated in iOS 14 and I think we'll get warnings eg in `ChatDropInteraction.dropInteraction(performDrop:)`

Reply:
Sorry I was not very clear. I just meant that everywhere in the app that uses `kUTTypeX` strings instead of `UTType.x` should be updated because the former is deprecated in iOS 14